.credo.exs

```diff
@@ -161,7 +161,15 @@
            order:
              ~w/shortdoc moduledoc behaviour use import alias require module_attribute defstruct callback/a
          ]},
-        {Credo.Check.Consistency.MultiAliasImportRequireUse, []},
+        {Credo.Check.Consistency.MultiAliasImportRequireUse,
+         files: %{
+           excluded: [
+             "lib/elixir_internal_certificate/account/accounts.ex",
+             "test/elixir_internal_certificate/account/accounts_test.exs",
```

Comment:
as we are following our convention, do we still need to skip this rule in 2 files? I think our structure should follow that rule already :D 

- "lib/elixir_internal_certificate/account/accounts.ex",
- "test/elixir_internal_certificate/account/accounts_test.exs",

Reply:
![image](https://user-images.githubusercontent.com/42540264/173278011-3478a99e-3c77-472f-a474-3534684272ed.png)
No it's not 👂